app/experimenter/targeting/constants.py

```diff
@@ -113,6 +113,33 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_REDUCED_MOTION = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default, preferring reduced motion"
+    ),
+    slug="first_run_new_profile_need_default_prefers_reduced_motion",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default, preferring reduced motion"
+    ),
+    targeting=(
+        "{first_run} && os.windowsBuildNumber >= 18362 && {new_profile} && "
+        "{need_default} && !userPrefersReducedMotion".format(
+            first_run=FIRST_RUN.targeting,
+            new_profile=NEW_PROFILE,
+            need_default=NEED_DEFAULT,
+        )
+    ),
+    desktop_telemetry=(
+        "{first_run} AND environment.system.os.windows_build_number >= 18362 AND "
+        "!isDefaultBrowser AND doesAppNeedPin && !userPrefersReducedMotion"
```

Comment:
I think check above should be same as targeting and should remove doesAppNeedPin, we can use this patch to fix desktop_telemetry clause in Line 108 and 109 which incorrectly includes doesAppNeedPin and leaves out new_profile